android/src/main/java/com/swmansion/rnscreens/ScreenContainer.java

```diff
@@ -377,4 +378,13 @@ protected void performUpdate() {
 
     tryCommitTransaction();
   }
+
+  protected void updateChildFragments() {
```

Comment:
There is nothing that we update in this method. maybe rename to `notifyOnUpdate` or `notifyContainerUpdate` or sth else along these lines